src/Checker/OptimizationChecker.php

```diff
@@ -166,7 +166,7 @@ public function getProperties(string $type, array $properties): array
 
     public function isHreflang(): bool
     {
-        return \count($this->getHreflang()) > 1;
+        return \count($this->getHreflang()) >= 1;
```

Comment:
@al2b : Is this change right, or was it expected somehow to be considered false if there is only one hreflang item? 🤔

Reply:
Change is right, there can be one or more hreflang items !